### ISSUE TITLE
Please give a reference to the procedure for saving passwords

### ISSUE THREAD

**Issue by RonCam** [QupZilla/qupzilla#1616]
I have checked [X] Allow saving passwords, but the table below it remains blank, as even though I have just before visited a site (tomorrow.do) that required a Username/Password entry.  

I am a LastPass user and so my only knowledge of saving passwords is that it 'just happens'.  For years I've been using Password Prompter, where the user carries out a manual select, copy-and-paste to get the information into the proper fields.  

Obviously, that's not the way it works in Qupzilla (at least, not so far) and I suspect there must be something I should do, to trigger this?  I can't find any documentation saying exactly what I should do, nor in looking at the main window do I see anything to click, or 'tic',  to trigger this action.

I have read the Qupzilla FAQs several times, and a search for 'password' on this page gets no 'hits'. In Qupzilla, I do see the button for importing passwords, but that's not how I want to get them into the program.

**Comment by JHooverman** [comment]
This is my working configuration:

![pw](http://i.imgur.com/xhQFMWW.png)

When I visit a site, where a login is provided, QupZilla asks me, if I would like to save the password. There is no advanced configuration required.

Do you use the plaintext or encrypted backend?

**Comment by RonCam** [comment]
Thanks so much for the reply.  Now I know what to look for.  

This will be -- a pop up window that appears over the web page?  And I assume, not only Password but Username would have to be entered by the user?  And then, after the last character is typed, one would hit Enter, and then a message box will become self-evident?

So sorry for asking what should be obvious -- but so far I've never seen this.  Now, knowing what to look for, let me try again, and let you know what I see.  

I see you have Allow Saving Passwords checked.  I'll make sure, but it was my impression that the default was the box being checked.  

Let me check the netbook with this information, and then I'll let you know what happened.

It's plaintext -- again, I think this was the default.  Until it started working, I didn't intend to change any of the default settings.

**Comment by JHooverman** [comment]
Tomorrow I will be able to provide a screenshot about the way saving of passwords work.

**Comment by RonCam** [comment]
Much appreciated if you could include in your example a login, and the saving of passwords, for this site:
https://www.tomorrow.do/
It's a free site, so you could easily create a 'practice account'.

I just logged-in manually, and I can't see any type of dialogue appearing, anywhere on the screen, asking me whether or not I want to save the Username/Password.  

Then, having done that, could you also illustrate the procedure for logging-in, hopefully with the Username and Password, now being provided by Qupzilla?

**Comment by JHooverman** [comment]
This is the way I use to know QupZilla's password-manager works:

Once you type in your username and password (example github.com), then hit login, QupZilla will ask to save the password:

![pw1](http://i.imgur.com/SeYmjZn.png)

But this dialog is not displayed on tomorrow.do.

@nowrep: Do you have an idea what might be the reason for this behaviour? What could be a solution?

**Comment by RonCam** [comment]
Thanks for checking on that.  I'd like for that page to open quickly, by accessing a stored Username and Password.  

Once this is resolved, there's also a matter of accessing a log-on page for access to free Wi-Fi hotspots, from my Internet service provider.  

That use is especially necessary, because until log-on to the hotspot is complete, Internet access is blocked, and without that, the LastPass bookmarklets won't work.  Of course, a LastPass extension would work, if it were possible to run that on Qupzilla.  

So, having access to a locally-stored Username/Passwork is the only solution I can think of -- unless Qupzilla has some other functionality that would allow this.

**Comment by RonCam** [comment]
Sorry, issue was closed in error -- probably hit the wrong button after editing the comment!  Much appreciated if nowrep has an answer to the immediate question.

**Comment by RonCam** [comment]
Sorry, JHooverman -- did I 'loose you' by accidentally closing this thread -- it was only a few hours before I discovered my mistake.  I'll wait another day or so, and then if you're still not back, I'll again close the thread and start another, hoping for fewer accidents on my part!

**Comment by nowrep** [comment]
The issue is probably the input being type=email. I'll test it myself.

**Comment by RonCam** [comment]
Thanks!  I'll wait.

**Comment by jf2944** [comment]
A website that doesn't work for me with saving passwords is "https://www.coursera.org" (MOOC platform). 

But the "https://github.com" website works as expected.

The problem with the coursera.org website may be related to the fact that it doesn't like any of the user-agent strings that I have tried configuring in the preferences->other->UserAgentManager menu. The website info indicates compatibility with "IE=Edge,chrome=IE7". 

![login-coursera](https://cloud.githubusercontent.com/assets/10554057/6765433/cfd4166a-cfb6-11e4-898b-c977b34ae3fc.png)

**Comment by jf2944** [comment]
I did manage to get the coursera.org web site to accept the UserAgent string by manually appending  "Chrome/41.0.2272.76" to the default string (i.e., no boxes checked) that was being sent.

But I still haven't been able to figure out why Qupzilla doesn't ask about saving the password for this particular site (other sites are OK).

**Comment by RonCam** [comment]
:pray: Bump?

**Comment by nowrep** [comment]
It doesn't work because both forms have empty `action` (in form element) and are sent using javascript.

**Comment by RonCam** [comment]
:disappointed:  OK, thanks for looking into it.  Not the fault of Qupzilla -- we would have to contact whoever coded the login for these sites, and ask them to use more conventional methods.
